Solutions/Marain.Tenancy.OpenApi.Service/Marain/Tenancy/OpenApi/TenancyService.cs

```diff
@@ -239,12 +240,19 @@ public class TenancyService : IOpenApiService
                 ITenant result = tenantId == RootTenant.RootTenantId
                     ? this.GetRedactedRootTenant()
                     : result = await this.tenantStore.GetTenantAsync(tenantId, etag).ConfigureAwait(false);
+
                 OpenApiResult okResult = this.OkResult(await this.tenantMapper.MapAsync(result).ConfigureAwait(false), "application/json");
+
                 if (!string.IsNullOrEmpty(result.ETag))
                 {
                     okResult.Results.Add("ETag", result.ETag!);
                 }
 
+                if (!string.IsNullOrEmpty(this.cacheConfiguration.GetTenantResponseCacheControlHeaderValue))
```

Comment:
No, you're right, it is inconsistent.

Ultimately we don't want it to be nullable. But because we have no way of knowing (as that property is populated via binding to configuration) then we should still check here.

Reply:
Having revisited this, I've made it more consistent by allowing the value to not be present, which means I no longer need the `#nullable disable` (because the property is now nullable). But I'm expecting it should be there, so I log a warning if it isn't.